actions/set_role_perms_MOD.js

```diff
@@ -230,28 +266,59 @@ module.exports = {
         result = 33554432n;
         break;
       case 29:
-        result = 1099511627775n;
+        result = 2199023255551n;
         break;
       case 30:
         result = 200n;
         break;
+      case 31:
+        result = 1024n;
+        break;
+      case 32:
+        result = 1099511627776n;
+        break;
+      case 33:
+        result = 274877906944n;
+        break;
+      case 34:
+        result = 34359738368n;
+        break;
+      case 35:
+        result = 68719476736n;
+        break;
+      case 36:
+        result = 137438953472n;
+        break;
+      case 37:
+        result = 17179869184n;
+        break;
+      case 38:
+        result = 2147483648n;
+        break;
+      case 39:
+        result = 549755813888n;
+        break;
+      case 40:
+        result = 8589934592n;
+        break;
       default:
         break;
     }
 
     if (role?.id) {
       const perms = role.permissions;
-      if (Array.isArray(role)) {
+
+      if (Array.isArray(role))
```

Comment:
This will never happen due to the if statement before this one (arrays don't have an `id` property)